stripe/src/main/java/com/stripe/android/model/Source.kt

```diff
@@ -67,8 +67,9 @@ data class Source internal constructor(
      * Set of key-value pairs that you can attach to an object. This can be useful for storing
      * additional information about the object in a structured format.
      *
-     * Note: This field will only be populated when retrieved using an ephemeral key.
+     * @deprecated Metadata is no longer returned to clients using publishable keys. Retrieve them on your server using you secret key instead.
      */
+    @Deprecated("Metadata is no longer returned to clients using publishable keys. Retrieve them on your server using you secret key instead.")
```

Comment:
"on your backend" vs "on your server"? Not sure the preferred wording.

"your secret" instead of "you secret"

Reply:
oh good catch. I literally just copy-pasted this from iOS so it will need to be updated there as well cc @csabol-stripe.

Don't have a strong opinion on backend vs server